Source/Scene/Model.js

```diff
@@ -4585,6 +4587,7 @@ define([
 
         this._rendererResources = undefined;
         this._cachedRendererResources = this._cachedRendererResources && this._cachedRendererResources.release();
+        DracoLoader.destroyCachedDataForModel(this);
```

Comment:
Should the cached data be reference counted, similar to `CachedRendererResources`? This scenario isn't working for me:

1. Load first model
2. Load second model
3. Destroy first model
4. Load third model

[localhost:8080 Sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=vVZdb9s2FP0rhF8sDSqlOE6LNW621EE7YA0WxEEfBr8w4rVFhCIFkrLjDf3vvSQlW/baLdhDBUMQqftx7rnnUt4wQzYCtmDIO6JgS+ZgRVvTz2EvGZdhOdfKMaHAjNPLpdqgjy1BAbpEXxqW+GqpVq0qndCKlAaYg1vNQSatkRmpQKwrlxHDFNf1nJUV/A67lPy9VASv+BpDdgg4rFgr3WcmW0h634IWHoG39yi0EWuhDj5zZhw+MXVOV0bXN7A2ADZ5dTY5p8Wb6fT12c8ZmU5pcVGcvyle95iGIWsP+JY5I54PcR8Qs11pU1taAeNCre+EK6t7LeWD/iCegX8wrIYk4smGRP52Yp+kaeDpKB9mCgzSxohaOLEBSxnnSRcjkBgq+ijdKukI8xcSS976e3bYG1bwdrga2JQd+2hw3A7yC0HjisbdJPXRFbYCm8+j+5e0pyveDbjWqJiHYs/57g6BCgvUVaCSXg9JMNh3uy+/RNoMOwgpri9PjFB+BrkLIo1EWZQXqEXDSpgHl/ne5sTZ+0xoQX6KhdXsOUKhjxorw9YsmgoMYMVctDbrECHbrXXIvQJm0kHIAxZaCyXqtv5To9KEdUyVfiIMZiroxWkJoFyA37c0dGRKa5S4aOTu/e5OC+U6aMOmkW+ijfGOlHYQf5KmJ+k71R7lr6jT97iN0k4m54PJ6p0ar9nvuryanLh0xEmtn65d8g2AR6PA1NqPddjJYqoscIe9OoI/1NdlL0GqUVxmiyo7CAyM0SYd6GsrUMNbyrBXrnu7D4APX/wYxtNsJYx1/ckG2GLeLVwlDA/zusCBKJl1EvxkPmgtH5l53zqHmcfXnMcY44zs8ey1PjwJx5Tm+99109h8wepGwg1zLA812vzGsFLPdd0YsBZ4Hum7FfLpwbTlE13jIYCJCszFpIX0PwctQMNGHnPoKUhfVlvk5EcWN6EXLy4vovu/9d1DrTfwb+3Tymp0lXqdBKv952Lw7Rue3CZEPNi+lOUgth9J8gV9uYYCuO9wPMpGM+t2Eq7i/q+ibrRx/ruUIEgHiA/x2/wRk4OjpbXec5b3TjMuNkTwd8vRyf+N5YiUklmLb1atlAvxFyxHV7Mc7Y/cpA4HyR8bMJLtvEl1dvUpblJKZzku/+nlYg8GEb8C)